### ISSUE TITLE
Can impute test set?

### ISSUE THREAD

**Issue by bbb801** [jwb133/InformativeCensoring#1]
Dear Sir,

I am wondering if the package can use the information trained from the training set and then be applied to the test set. Or, I should repeat the imputation twice respective on the training and test set. Thanks.

Best,
Beato
